python/ray/tests/test_object_manager.py

```diff
@@ -647,7 +647,8 @@ def f(x):
     start = time.time()
     ray.get(remote_tasks)
     end = time.time()
-    assert end - start < 10, "Too much time spent in pulling objects, " \
+    # end - start would be greater than 20 if there is a retry.
+    assert end - start < 19, "Too much time spent in pulling objects, " \
```

Comment:
why <19 and not < 20?

Reply:
my bad. you explain it in the description..

Reply:
Yeah 20s would probably still work.